modules/cluster-logging-collector-log-forwarding-supported-plugins.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins_{context}"]
+
+= Supported plugins for forwarding logs to third-party systems
+
+[role="_abstract"]
+Red Hat OpenShift Logging version 5.0 tests and supports the following plugins for forwarding logs to third-party systems.
+
+.Plugins
+[options="header"]
+|====
+|Output type|Plugin name|Plugin version|Tested with
+
+| Elasticsearch | link:https://github.com/uken/fluent-plugin-elasticsearch[fluent-plugin-elasticsearch] | link:https://github.com/uken/fluent-plugin-elasticsearch/releases/tag/v4.1.1[4.1.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/elasticsearch.go#L219[Elasticsearch 6.8.1]
+
+| Fluent forward protocol | link:https://github.com/fluent/fluentd[fluentd] | link:https://github.com/fluent/fluentd/releases/tag/v1.7.4[1.7.4] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/fluentd/receiver.go#L96[fluent forward v1]
+
+| syslog | link:https://github.com/docebo/fluent-plugin-remote-syslog[fluent-plugin-remote-syslog] | link:https://rubygems.org/gems/fluent-plugin-remote-syslog[1.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/e2e/logforwarding/syslog/forward_to_syslog_test.go#L156[RFC-5424], link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/e2e/logforwarding/syslog/forward_to_syslog_test.go#L391[RFC-3164]
+
+| Apache Kafka | link:fluent-plugin-kafka[fluent-plugin-kafka] | link:https://github.com/fluent/fluent-plugin-kafka/releases/tag/v0.13.1[0.13.1] | link:https://github.com/openshift/cluster-logging-operator/blob/release-5.0/test/helpers/kafka/broker.go#L141[kafka 2.4.1]
+
+|====
+
+[role="_Where"]
+.Where
+
+Output type:: The target log collector type or protocol.
+Plugin:: The name of the tested plugin or component, which also links to the plugin website.
+Plugin version:: The version of the tested plugin or component, which also links to the version information on the plugin website or link:https://rubygems.org/gems/[rubygems.org]
+Tested with:: The target log collector, protocol, or standard, and their versions, with which Red Hat tests the plugin. These items link to link:https://github.com/openshift/origin-aggregated-logging/blob/release-5.0/fluentd/Gemfile.lock[the corresponding line] of the test configuration file that Red Hat OpenShift Logging version 5.0 uses. This file is the authoritative source of information about which items and versions Red Hat tests and supports.
+
+[NOTE]
+====
+Although the plugins in the preceding table support a more comprehensive range of target logging collector types and versions, Red Hat tests and supports only the items shown in "Tested with" column.
```

Comment:
I would probably frame this a bit different since "supports" might be to hard a cut at the moment. We should use some better language to not have another "panic" :) How about we only say that we test and therefore have higher confidence that it is actually working?!

Reply:
Thanks. Updated.